src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
- `(click)`, `(keydown.enter)` and other `@Output` parameters should be binding only with existing methods or using `(click)="onClick.emit($event)"` syntax.

Please, in future try to build the ng2-smart-table demo with AOT before pull-request.